addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -346,8 +346,17 @@ def _process_step_forward_operator(self, discuss_channel):
                 # first post the message of the step (if we have one)
                 posted_message = discuss_channel._chatbot_post_message(self.chatbot_script_id, plaintext2html(self.message))
 
-            # next, add the human_operator to the channel and post a "Operator joined the channel" notification
-            discuss_channel.with_user(human_operator).sudo().add_members(human_operator.partner_id.ids, open_chat_window=True)
+            # next, add the human_operator to the channel
+            discuss_channel.with_user(human_operator).sudo().add_members(human_operator.partner_id.ids, open_chat_window=True, post_joined_message=False)
+            # post a "Operator joined the channel" notification
+            discuss_channel.message_post(
```

Comment:
This can be removed in the master branch and override just `post_joined_message` to get the same result and avoid manually logging the message.
See https://github.com/odoo/odoo/pull/181150#discussion_r1891202955